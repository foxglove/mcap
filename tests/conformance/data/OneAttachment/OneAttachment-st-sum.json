{
  "records": [
    {"type": "Header", "fields": [["library", ""], ["profile", ""]]},
    {
      "type": "Attachment",
      "fields": [
        ["create_time", "1"],
        ["data", ["1", "2", "3"]],
        ["log_time", "2"],
        ["media_type", "application/octet-stream"],
        ["name", "myFile"]
      ]
    },
    {"type": "DataEnd", "fields": [["data_section_crc", "3051995293"]]},
    {
      "type": "Statistics",
      "fields": [
        ["attachment_count", "1"],
        ["channel_count", "0"],
        ["channel_message_counts", {}],
        ["chunk_count", "0"],
        ["message_count", "0"],
        ["message_end_time", "0"],
        ["message_start_time", "0"],
        ["metadata_count", "0"],
        ["schema_count", "0"]
      ]
    },
    {
      "type": "SummaryOffset",
      "fields": [
        ["group_length", "55"],
        ["group_opcode", "11"],
        ["group_start", "116"]
      ]
    },
    {
      "type": "Footer",
      "fields": [
        ["summary_crc", "2285786167"],
        ["summary_offset_start", "171"],
        ["summary_start", "116"]
      ]
    }
  ],
  "meta": {"variant": {"features": ["st", "sum"]}}
}
