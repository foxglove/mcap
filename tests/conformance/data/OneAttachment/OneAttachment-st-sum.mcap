version https://git-lfs.github.com/spec/v1
oid sha256:d9f0eaf6d251cc4ad4677daca3f01ad8c8793e7d3966cd68d1ec0f25ca8f0050
size 264
