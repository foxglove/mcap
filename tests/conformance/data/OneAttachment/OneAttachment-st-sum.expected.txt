Header library=s:"" profile=s:""
Attachment content_type=s:application/octet-stream created_at=n:1 data=b:<010203> log_time=n:2 name=s:myFile
Statistics attachment_count=i:1 channel_count=i:0 channel_message_counts=m:{} chunk_count=i:0 message_count=n:0
SummaryOffset group_length=n:0 group_opcode=i:13 group_start=n:149
SummaryOffset group_length=n:33 group_opcode=i:11 group_start=n:116
SummaryOffset group_length=n:0 group_opcode=i:8 group_start=n:149
Footer summary_crc=i:0 summary_offset_start=n:149 summary_start=n:116
